### ISSUE TITLE
The return value of predict_proba function in causality.pairwise

### ISSUE THREAD

**Issue by NawTun** [FenTechSolutions/CausalDiscoveryToolbox#98]
Dear Diviyan:
I found that in doc 
![image](https://user-images.githubusercontent.com/46189446/114964903-353cce00-9ea2-11eb-80a1-f1164ce8a188.png)
Does the return value of this function is between -1 and 1?
but when I applied this function I often got some much larger values such as 13,14, 
do this larger values mean that the causal relation is much stronger?
Thank you for your reply.

**Comment by Diviyan-Kalainathan** [comment]
Hello @NawTun,

It depends on the algorithm, but currently `,predict_proba` outputs the raw output of the algorithm. Which one are you using ? 

Best,
Diviyan

**Comment by NawTun** [comment]
Hi @Diviyan-Kalainathan,

Thanks for your reply, I used **ANM** algorithm, 


Although this function returns a raw output, I can still compare the absolute value one output to another output in order to understand relative probability. 
For example, `predict_proba(a,b)` returns 4 when `predict_proba(c,d)` returns 2,
so a->b is relatively more possible than c->d ,

am I right?

**Comment by Diviyan-Kalainathan** [comment]
Yes that is correct. The algorithm is more confident in a -> b than c ->d. Did you check with the original (matlab) implementation of ANM ?  Those values does seem high for ANM...

**Comment by NawTun** [comment]
OK, Thanks for your affirmance, Diviyan,
Maybe the original implementation can help to understand, I will check it later.
In addition, CDT is really a great tool, :blush: 
Thanks again.

**Comment by 945716994** [comment]
If the value is between -1 and 1, doesn't that mean it doesn't know the direction of the edge.
